dts/bindings/spi/nordic,nrf-exmif.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
Why is this commit in this PR? This change was already merged to sdk-zephyr main: https://github.com/nrfconnect/sdk-zephyr/commit/7ea6937f02affb74cae2327aeca955ae5e5c3106

Reply:
The EXMIF patch is reverted and re-applied to avoid conflicts.